log/log.go

```diff
@@ -38,6 +39,27 @@ func Init(cfg Config) {
 	}
 }
 
+// InitTestLogger initializes a logger for testing purposes. The returned logs
+// can be filtered against test assertions.
+// Call DeinitTestLogger to restore the default logger.
+func InitTestLogger(cfg Config) (*observer.ObservedLogs, error) {
+	var (
+		err          error
+		observedLogs *observer.ObservedLogs
+	)
+	log, observedLogs, err = NewTestLogger(cfg)
+	if err != nil {
+		return nil, err
+	}
+	return observedLogs, nil
+}
+
+// DeinitTestLogger deinitializes the test logger and restore the default one.
+func DeinitTestLogger() {
+	log = nil
+	getDefaultLog()
+}
+
```

Comment:
It is ok to see the logs. To avoid problems check how to initialize logs at the beginning of your test. You can take a look here.
https://github.com/0xPolygonHermez/zkevm-node/blob/717270900a03987a3b4c9e71736f00dbaf964c73/etherman/etherman_test.go#L23

Reply:
@ARR552 I think the comment for `InitTestLogger` was a bit misleading and I slightly modified it here a927ba45b7a6b3899c04312dd09d14e637f09dfc.

This logger is meant to be used in tests where you need to inspect the logs. You can see how I used it in this test: 
https://github.com/0xPolygonHermez/zkevm-node/blob/033877649b2fb3a088ea957b22c91fef656aaa40/ethtxmanager/ethtxmanager_test.go#L45

Anyway the latest version of that test has changed (see 85221c0d861788aef453b8e161ea722bdd590590) because, after rebasing on `develop`, the signature of the tested method has changed. Now it returns an error and so inspecting the logs is not required any more.
So my test-logger is not used any more, but I'd keep it in any case, it may become useful for other tests, WDYT?

I haven't found a way to use our default logger for this purpose (it wants either a URL or a file path for outputting the logs. I'm not sure a buffer can fit there)

I hope this clarifies a bit.

Reply:
If it is not used I would preffer remove it just to avoid have unused code. We can always add it again when is required

Reply:
Fair enough @ARR552 , removed here 7221252b96505e4cbf812ef36c61b6818cb0e66d